### ISSUE TITLE
How do we know what belongs to X and what belongs to W?

### ISSUE THREAD

**Issue by itewqq** [microsoft/EconML#656]
Hi there,

I have noted the following discussion here: https://github.com/microsoft/EconML/issues/589

What I'm curious about is, in a real scenario, for some FEATURES, I can't know which ones have an effect on T and which ones don't, so how should I divide X and W at this point? Thanks!

**Comment by salman-moh** [comment]
Regarding bad effect on final result, if you cant distinguish W and X, which I assume are the sufficient adjustment set and set of all features, then you going to have an incorrect causal estimation.
For my understanding how you go from all set of features X -> to specific set of features W is by domain knowledge understanding.

**Comment by kbattocchi** [comment]
W and X should both be things that affect T and Y, the difference is that things in X are allowed to also affect the _strength_ of the relationship between T and Y.  In general, if you don't know whether something might affect that relationship or not, it's probably safer to default to including it in X rather than in W.  However, the downside of this is that this makes the treatment effect estimation problem harder, so you should expect to get wider confidence intervals on the effect estimate even if it turns out that that particular feature does not affect the strength of the relationship.

**Comment by salman-moh** [comment]
so then are X and W together all just confounder variables?

**Comment by kbattocchi** [comment]
Yes, exactly.  And for techniques like Double ML, we concatenate X and W together for fitting our first stage models, but then we only featurize and interact X with the T residuals when fitting the second stage model.

**Comment by justforsoy** [comment]
What if a feature do not direct affect Y and T,  but it affect the strength of the relationship between T and Y.
Should I add it to X?

**Comment by kbattocchi** [comment]
@justforsoy If it affects the relationship between T and Y, then I think it must inherently affect Y as a result of this relationship (i.e. most of our estimators assume the structural model Y=theta(X)*T+..., so Y does vary based on X even if X does not appear in the remainder of the expression).

But to directly answer your question, yes, a variable should go into X if it affects the strength of the relationship.
